docs/tutorials/01_introductory/viz_solar_system.py

```diff
@@ -0,0 +1,309 @@
+
+r_mercury = 7
+r_venus = 9
+r_earth = 11
+r_mars = 13
+r_jupiter = 16
+r_saturn = 19
+r_uranus = 22
+r_neptune = 25
+
+g_exponent = np.float_power(10, -11)
+g_constant = 6.673*g_exponent
+
+m_exponent = np.power(10, 30)
+m_constant = 1.989*m_exponent
+
+miu = m_constant*g_constant
+
+##############################################################################
+# Let's define two functions that will help us calculate the position of each
+# planet as it orbits around the sun: ``get_orbit_period`` and
+# ``get_orbital_position``, using the constant miu and the orbital radii
+# of each planet.
+
+
+def get_orbit_period(radius):
+    return np.sqrt(2*np.pi * np.power(radius, 3)/miu)
+
+
+def get_orbital_position(radius, time):
+    orbit_period = get_orbit_period(radius)
+    x = radius * np.cos((2*np.pi*time)/orbit_period)
+    y = radius * np.sin((2*np.pi*time)/orbit_period)
+    return (x, y)
+
+
+##############################################################################
+# Let's change the camera position to visualize the planets better.
+
+scene.set_camera(position=(-20, 60, 100))
+
+##############################################################################
+# Next, create a ShowManager object. The ShowManager class is the interface
+# between the scene, the window and the interactor.
+
+showm = window.ShowManager(scene,
+                           size=(900, 768), reset_camera=False,
+                           order_transparent=True)
+
+##############################################################################
+# Next, let's focus on creating the animation.
+# We can determine the duration of animation with using the ``counter``.
+# Use itertools to avoid global variables.
+
+counter = itertools.count()
+
+##############################################################################
+# To track and visualize the orbital paths of the planets, we will create
+# several new variables to map each planet's orbits using ``actor.dots``.
+# It is important to define the track variable for each planet as global,
+# allowing it to be used within the ``timer_callback`` function. To do this,
+# create a function called ``get_orbit_actor``.
+
+
+def get_orbit_actor(orbit_points):
+    orbit_actor = actor.dots(orbit_points, color=(1, 1, 1),
+                             opacity=1, dot_size=1)
+    return orbit_actor
+
+
+##############################################################################
+# All of the planets will have the same initial positions, so assign each of
+# those to the positions variables for each planet. These variables will be
+# updated within the ``timer_callback`` function. Initialize and add the
+# orbit actors into the scene. Also initialize the track variables for each
+# planet.
+
+orbit_points = np.zeros((2100, 3), dtype='f8')
+
+mercury_orbit_actor = get_orbit_actor(orbit_points)
+scene.add(mercury_orbit_actor)
+positions_mercury = utils.vertices_from_actor(mercury_orbit_actor)
```

Comment:
None of the orbits are being rendered when I run the code.

Reply:
@lenixlobo, I just committed a change to the timer_callback function to make the rendering of the orbit paths occur earlier in the animation. Let me know if you are able to see them now. Thank you!

Reply:
As you can see in the attached screenshot I am not able to see the orbit

![image](https://user-images.githubusercontent.com/20311706/87329387-7a6b5280-c554-11ea-89ca-3d6a48b670de.png)

Reply:
the orbit arrives later in the timer @lenixlobo? Are you running the tutorial until the end?

Reply:
They weren't in the previous commit, but now its working.